src/Mews.Fiscalizations.All.sln

```diff
@@ -52,6 +52,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Mews.Fiscalizations.Basque"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Mews.Fiscalizations.Basque.Tests", "Basque\Mews.Fiscalizations.Basque.Tests\Mews.Fiscalizations.Basque.Tests.csproj", "{9A27DA76-278F-4B87-8AE3-0504C09AA1B8}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Bizkaia", "Bizkaia", "{B33E5A1D-A667-477D-9AD8-1673192B71BB}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Mews.Fiscalizations.Bizkaia", "Mews.Fiscalizations.Bizkaia\Mews.Fiscalizations.Bizkaia.csproj", "{145C287C-2D14-4B35-A73A-6A97ABBF3F10}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Mews.Fiscalizations.Bizkaia.Tests", "Mews.Fiscalizations.Bizkaia.Tests\Mews.Fiscalizations.Bizkaia.Tests.csproj", "{2767CE08-C4C2-4043-AC58-C413ECB64D3A}"
```

Comment:
If i remember correctly, Bizkaia and the other 2 regions share most of the stuff (almost identical XSDs)

so i suggest that we either implement Bizkaia within the same project (Basque), or rename the Basque project to something else like GipuzkoaAndAraba, or having a project per region.

I would go with the first solution (all regions within 1 project).

Reply:
Ok, I will go with option 1 and have all regions in the same project..